tools/build-tools/src/typeValidator/testGeneration.ts

```diff
@@ -87,14 +87,25 @@ interface TestCaseTypeData extends TypeData{
 function buildTestCase(getAsType:TestCaseTypeData, useType:TestCaseTypeData, isCompatible: boolean){
     const getSig =`get_${getAsType.prefix}_${getFullTypeName(getAsType).replace(".","_")}`;
     const useSig =`use_${useType.prefix}_${getFullTypeName(useType).replace(".","_")}`;
+    const getIsRemoved = getAsType.kind.startsWith("Removed");
+    const useIsRemoved = useType.kind.startsWith("Removed");
     const testString: string[] =[];
-    testString.push(`declare function ${getSig}():\n    ${toTypeString(getAsType.prefix, getAsType)};`);
-    testString.push(`declare function ${useSig}(\n    use: ${toTypeString(useType.prefix, useType)});`);
+
+    testString.push(`declare function ${getSig}():`);
+    if (!isCompatible && getIsRemoved) {
+        testString.push(`    // @ts-expect-error compatibility expected to be broken`);
```

Comment:
it we reused the same comment string its worth it to put it in a constant somewhere